melodies_monet/__init__.py

```diff
@@ -1,12 +1,38 @@
 """
 MELODIES MONET
 """
-from . import driver, plots, stats, util, tutorial
+import sys
 
-__all__ = (
+__version__ = "0.1.dev"
```

Comment:
For main we should probably remove the dev. How about 0.1.0? @rschwant

Reply:
Keeping dev to indicate development release/version is fine too

Reply:
Let's keep dev for development for now. Once we have an official release to coordinate with publication of a paper we can take this off.

Reply:
Ok, sounds good.